packages/fields/src/types/Select/test-fixtures.js

```diff
@@ -53,13 +71,77 @@ export const filterTests = withKeystone => {
     )
   );
 
+  test(
+    'No filter (dataType: string)',
+    withKeystone(({ keystone }) =>
+      match(
+        keystone,
+        undefined,
+        [
+          { name: 'a', selectString: 'a string' },
+          { name: 'b', selectString: '@¯\\_(ツ)_/¯' },
+          { name: 'c', selectString: 'a string' },
+          { name: 'd', selectString: '1number' },
+        ],
+        'name selectString'
+      )
+    )
+  );
+
+  test(
+    'No filter (dataType: number)',
+    withKeystone(({ keystone }) =>
+      match(
+        keystone,
+        undefined,
+        [
+          { name: 'a', selectNumber: 1 },
+          { name: 'b', selectNumber: 2 },
+          { name: 'c', selectNumber: 3 },
+          { name: 'd', selectNumber: 1 },
+        ],
+        'name selectNumber'
+      )
+    )
+  );
+
   test(
     'Filter: company',
```

Comment:
Can we add `(dataType: enum)` on all of the original tests just for consistency?